src/main/java/uk/gov/companieshouse/lookup/internationalisation/SessionProvider.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.lookup.internationalisation;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.session.Session;
+import uk.gov.companieshouse.session.handler.SessionHandler;
+
+import java.util.Map;
+
+// Having an instance of this class allows to it to be mocked
```

Comment:
I don't think you need this comment, this is a perfectly legitimate class

Reply:
Removed